src/test/java/de/retest/recheck/testng/ExecuteTest.java

```diff
@@ -13,27 +13,23 @@
 
 	private static class DummyClass {
 
-		private RecheckLifecycle someField;
-		private RecheckLifecycle otherField;
+		@SuppressWarnings( "unused" )
+		public final Object someObject = new Object();
+		protected final RecheckLifecycle someLifecycle = mock( RecheckLifecycle.class );

```

Comment:
Why is this field `protected`?

Reply:
I thought I play around with different field visibilities for testing. Do you find it confusing?

Reply:
I was a bit confused if it was an accident or not. The intention to have different visibility levels for testing could be reflected in the variable names: `publicObject`, `protectedLifecycle` and `privateLifecycle`.

Reply:
Good idea, let me change that!